core/src/main.py

```diff
@@ -50,8 +50,7 @@ def _boot_default() -> None:
     if __debug__:
         import apps.debug
     if not utils.BITCOIN_ONLY:
-        if not __debug__ or utils.EMULATOR:
-            import apps.webauthn
+        import apps.webauthn
```

Comment:
why not move this up to the other `not BITCOIN_ONLY` block? do we need apps.debug first?

Reply:
I'm unsure. I didn't touch the other code on purpose. Maybe it's just a leftover from the previous code: https://github.com/trezor/trezor-firmware/blame/07ef01f3e498449d7f32dc5df898897ed877d60b/core/src/main.py#L69

I'll try it and run tests.